### ISSUE TITLE
<rocket-navigation> a11y

### ISSUE THREAD

**Issue by bennypowers** [daKmoR/rocket#14]
`<rocket-navigation>` has a click listener, but doesn't have interactive semantics:

```js
    this.addEventListener('click', ev => {
      if (ev.target.classList.contains('anchor')) {
        this.dispatchEvent(new Event('close-overlay', { bubbles: true }));
      }
    });
```

possible solutions:
- add listeners to `<a>` children instead of the parent

**Comment by daKmoR** [comment]
why would this be a warning? 🤔

event delegation is so much more performant and simpler to maintain then individual event listener... 

can you link some details to this rule or post a screenshot of the a11y result or how to reproduce it - I'm a little baffled 😱

**Comment by bennypowers** [comment]
This report comes from by Firefox' [accessibility inspector](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector) Fire up FF Developer Edition, load a rocket page (in desktop resolution) and then load any page. Switch to the accessibility view (right click an element and inspect accessibility properties) Then click the dropdown in a11y panel and scan for _keyboard_ issues.


The error is reported on the `<rocket-navigation>` element.

<img width="285" alt="Screen Shot 2020-08-10 at 9 49 54" src="https://user-images.githubusercontent.com/1466420/89757762-de028480-daee-11ea-9eef-be1f2c302472.png">

The specific error reported is [Clickable elements must be focusable and should have interactive semantics](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard#Clickable_elements_must_be_focusable_and_should_have_interactive_semantics)

> If an element can be clicked with a pointing device, such as a mouse, then it should also be focusable using the keyboard, and the user should be able to do something by interacting with it.
> 
> An element is clickable if it has an onclick event handler defined. You can make it focusable by adding a tabindex=0 attribute value to it. You can make it operable with the keyboard by defining an onkeydown event handler; in most cases, the action taken by event handler should be the same for both types of events.

aXe doesn't seem to report this specific error.
